components/Image/image.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image</title>
+    <link rel="stylesheet" href="image.css" />
+  </head>
+  <body>
+    <div class="container-grid">
+      <header class="header">
+        <h1 class="logo"><a href="#">RapidUI</a></h1>
+
+        <div class="header-nav">
+          <nav class="nav">
+            <ul>
+              <li><a href="#">Home</a></li>
+              <li><a href="#">Documentation</a></li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <aside>
+        <hr>
+        <div class="sidebar">
+          <h4>Components</h2>
+          <ul>
+            <li><a href="/components/Avatar/avatar.html" class="remove-deco">Avatar</a></li>
+            <li><a href="/components/Alert/alert.html" class="remove-deco">Alert</a></li>
+            <li><a href="/components/Badge/badge.html" class="remove-deco">Badge</a></li>
+            <li><a href="/components/Button/button.html" class="remove-deco">Button</a></li>
+            <li><a href="#" class="remove-deco">Card</a></li>
+            <li class="active"><a href="/components/Image/image.html" class="active-color remove-deco">Image</a></li>
+            <li><a href="#" class="remove-deco">Input</a></li>
+            <li><a href="/components/Text Utilities/text_utilities.html" class="remove-deco">Text Utilities</a></li>
+            <li><a href="/components/Navigation/navigation.html" class="remove-deco">Navigation</a></li>
+          </ul>
+        </div>
+      </aside>
+      <main>
+        <h1>Image</h1>
+        <p class="main-para">
+          Images provide visual information of an application.
+        </p>
+        <h1>Responsive Image</h1>
+        <div class="container-image">
+          <img src="/components/images/responsive-image.jpg" alt="responsive-image" class="img-resp">
+        </div>
+
+        <h1>Round Image</h1>
+        <br>
+        <br>
+        <img src="/components/images/responsive-image.jpg" alt="responsive-image" class="img-round">
+
+        <iframe
+          src="https://carbon.now.sh/embed/6sofNhu8BVaNrTKYnusb"
+          style="width: 680px; height: 353px; border:0; transform: scale(1); overflow:hidden;"

```

Comment:
It was mentioned in LMS that we must avoid using inline styling.
So this sizing can be shifted to the stylesheet.

Reply:
Hey so this is actually taken from a external website carbon.now.sh to display code in prettier way. So it was advisable not to change stuff from it, hence i have put it as it is. Rest i have not used inline styling anywhere